### PR TITLE
Replace icondir with an url schema

### PR DESCRIPTION
**Pull request by hellcp**

### CODE REVIEW

File: src/lib/packager/clients/repositories.rb [hunk 1]
```diff
@@ -355,10 +355,8 @@ def repoInfoRichText(name, category, info)
         Ops.get_string(URL.Parse(url), "scheme", "")
       )
       icon_tag = Ops.add(
-        Ops.add(
-          Ops.add(Ops.add("<IMG SRC=\"", Directory.icondir), "/22x22/apps/"),
-          ["cd", "dvd", "iso"].include?(schema) ? "yast-cd_update.png" : "yast-http-server.png"
-        ),
+        Ops.add("<IMG SRC=\"icon://22/\"",
```
Comment (lslezak) [inline src/lib/packager/clients/repositories.rb]: There is one extra double quote (`"`) at the end.

I think to make it better readable the whole block should be completely rewritten:

```ruby
icon = ["cd", "dvd", "iso"].include?(schema) ? "yast-cd_update.png" : "yast-http-server.png"
icon_tag = "<IMG SRC=\"icon://22/#{icon}\">&nbsp;&nbsp;&nbsp;"
```

File: src/include/checkmedia/ui.rb [hunk 2]
```diff
@@ -75,13 +75,8 @@ def TranslateInfo(info)
...
           # rich text message, %1 = CD identification
           "app"        => Ops.add(
-            Ops.add(
-              Ops.add(
-                Ops.add("<P><IMG SRC=\"", Directory.icondir),
-                "/22x22/apps/yast-checkmedia.png\">&nbsp;&nbsp;&nbsp;"
-              ),
-              _("<BIG><B>%1</B></BIG>")
-            ),
+            Ops.add("<P><IMG SRC=\"icon://22/yast-checkmedia\">&nbsp;&nbsp;&nbsp;",
```
Comment (lslezak) [inline src/include/checkmedia/ui.rb]: You can replace the `Ops.add` by simple `+` operator here.

File: src/lib/packager/clients/repositories.rb [hunk 3]
```diff
@@ -354,13 +354,8 @@ def repoInfoRichText(name, category, info)
...
         Ops.get_string(URL.Parse(url), "scheme", "")
       )
-      icon_tag = Ops.add(
-        Ops.add(
-          Ops.add(Ops.add("<IMG SRC=\"", Directory.icondir), "/22x22/apps/"),
-          ["cd", "dvd", "iso"].include?(schema) ? "yast-cd_update.png" : "yast-http-server.png"
-        ),
-        "\">&nbsp;&nbsp;&nbsp;"
-      )
+      icon = ["cd", "dvd", "iso"].include?(schema) ? "yast-cd_update.png" : "yast-http-server.png"
```
Comment (lslezak) [inline src/lib/packager/clients/repositories.rb]: Um, the other changes above actually removed the `.png` suffix. Do we still need it here?

Comment (hellcp) [reply src/lib/packager/clients/repositories.rb]: Ah, damn it

**Pull request merged**